interface/ceed-vector.c

```diff
@@ -493,6 +493,103 @@ int CeedVectorNorm(CeedVector vec, CeedNormType norm_type, CeedScalar *norm) {
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Compute y = alpha x + y
+
+  @param y[in/out]  target vector for sum
```

Comment:
```suggestion
  @param y[in,out]  target vector for sum
```
This is what we use elsewhere and I've seen documented.